src/styles/reset.scss

```diff
@@ -0,0 +1,6 @@
+// Style normalizujące CSS
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
Normalize może byłoby bardziej uniwersalną nazwą dla tego pliku?

Reply:
Obojętne, można :)

Reply:
Ok to tylko sugestia :) albo zmienisz albo nie :)